src/update.c

```diff
@@ -395,8 +395,14 @@ static int main_update()
 	ret = add_included_manifests(server_manifest, &latest_subs);
 	timelist_timer_stop(global_times);
 	if (ret) {
-		ret = EMANIFEST_LOAD;
-		goto clean_exit;
+		if (ret == -add_sub_BADNAME) {
```

Comment:
I know these definitions are already merged, but it feels odd to have constant names mix case.

Reply:
true